src/v2i-hub/PedestrianPlugin/manifest.json

```diff
@@ -32,6 +32,29 @@
 			"key":"WebServicePort",
 			"default":"9000",
 			"description":"Port at which Web service exists"
+		},
+		{
+			"key":"DataProvider",
+			"default":"FLIR",
+			"description":"Possible data providers we handle - FLIR or PSM XML data"
+		},	
+		{
+			"key":"WebSocketIP",
+			"default":"192.168.55.183",
+			"description":"IP address of the camera to connect to for data"
+
+		},
+		{
+			"key":"WebSocketURLExt",
+			"default":"80",
+			"description":"Extension/port for FLIR camera subscription"
+		},
+		{
+			"key":"FLIRCameraRotation",
+			"default":"16.13",
+			"description":"Measured camera rotation used for heading conversion"
```

Comment:
A little more description here may help. Units and relative to what?

Reply:
Done